### PR TITLE
matomo: 4.10.1 -> 4.14.2

### PR DESCRIPTION
**Pull request by boozedog**
###### Description of changes

Upgrades matomo from 4.10.1 to 4.14.2

Includes @twey's changes from https://github.com/NixOS/nixpkgs/pull/218989

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by MatthewCroughan — APPROVED** [review]
`nix build github:boozedog/nixpkgs/matomo-4.14.2#nixosTests.matomo -L` seems to pass

**Comment by MatthewCroughan** [comment]
@boozedog you need to change your commits so that they match the contributing guide. A single commit with the same title as the PR will suffice.

**Comment by boozedog** [comment]
> @boozedog you need to change your commits so that they match the contributing guide. A single commit with the same title as the PR will suffice.

Done. Thanks for the feedback!

**Comment by boozedog** [comment]
I've got it running in production and it's working fine

image

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2222,6 +2222,13 @@
     githubId = 68566724;
     name = "bootstrap-prime";
   };
+  boozedog = {
+    name = "David A. Buser (boozedog)";
```
Comment (drupol) [inline maintainers/maintainer-list.nix:2226]:
```suggestion
    name = "David A. Buser";
```

**Review by drupol — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (drupol) [reply maintainers/maintainer-list.nix:2226]: Could you also please sort the attributes alphabetically?

Comment (boozedog) [reply maintainers/maintainer-list.nix]: > Could you also please sort the attributes alphabetically?

Done.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -2222,6 +2222,13 @@
     githubId = 68566724;
     name = "bootstrap-prime";
   };
+  boozedog = {
+    email = "code@booze.dog";
+    github = "boozedog";
+    githubId = 1410808;
+    matrix = "@boozedog:matrix.org";
+    name = "David A. Buser (boozedog)";
```
Comment (drupol) [inline maintainers/maintainer-list.nix:2230]:
```suggestion
    name = "David A. Buser";
```
Your pseudo is already in there, there's no need to duplicate it.

**Review by drupol — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (boozedog) [reply maintainers/maintainer-list.nix]: Fixed.

**Comment by boozedog** [comment]
@drupol per our discussion I've re-added the matomo-beta version. Interested to hear from @twey if there's any concern here.

Re-tested using
```sh
nix build .#nixosTests.matomo.matomo
nix build .#nixosTests.matomo.matomo-beta
```

I've got the non-beta version running live on production:

image

**Review by drupol — APPROVED** [review]
LGTM

**Review by drupol — APPROVED** [review]
LGTM

**Comment by Twey** [comment]
@boozedog no problem at all, I just couldn't find it in the releases so I figured it had been dropped — I guess you found one :)

**Pull request merged**